README.md

```diff
@@ -1,67 +1,68 @@
 # Alfred API
 
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](
-  https://www.gnu.org/licenses/lgpl-3.0)
-[![Maven Central](https://img.shields.io/maven-central/v/eu.xenit.apix/apix-interface.svg)](
-  http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22eu.xenit.apix%22%20AND%20a%3A%22apix-interface%22)
-[![Jenkins Build Status](https://jenkins-2.xenit.eu/buildStatus/icon?job=Xenit+Github%2Falfred-api%2Fmaster&subject=Jenkins)](
-  https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-api/job/master/)
-
-## Rules for pull requests
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Jenkins Build Status](https://jenkins-2.xenit.eu/buildStatus/icon?job=Xenit+Github%2Falfred-api%2Fmaster&subject=Jenkins)](https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-api/job/master/)
+[![Maven Central](https://img.shields.io/maven-central/v/eu.xenit.apix/apix-interface.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22eu.xenit.apix%22%20AND%20a%3A%22apix-interface%22)
+
+# Documentation
+
+## Full
+
+[https://docs.xenit.eu/alfred-api/stable-user/index.html](https://docs.xenit.eu/alfred-api/stable-user/index.html)
+
+## Quickstart
+
+### Rules for pull requests
 * Common sense trumps all rules.
-* For every pull request please extend [CHANGELOG.md](./CHANGELOG.md).
+* For every pull request please extend the [CHANGELOG.md](./CHANGELOG.md).
 * Do not make breaking changes since this is an API used by customers. Breaking changes include 
   adding, changing or removing endpoints or JSON objects used in requests and responses.
   * If you are forced to make a breaking change:
     * Notify maintainers
     * Add a note to the changelog with upgrade instructions
     * Notify all customers at the next release
 * When working in REST code, please comply to **REST HTTP result codes** policy outlined in the
-  [developer guide](./developer-documentation).
+  [user guide](https://docs.xenit.eu/alfred-api/stable-user/rest-api/index.html#rest-http-result-codes).
 * Prefer unit tests over integration tests to keep builds fast
-* Follow our 
```

Comment:
Can you put this item back?

Reply:
Just the last point about the `this` keyword? or the link to the XEP too? Because that one is not public. I also checked the code style repo, and that one is also not public.

Reply:
Both actually (I misread that it was one). 
You're remark is correct, it is not public. Yes, we'll have to do something about that. But since now contributors will be 99.99% Xenit, let's keep it the way it was for now.